test/k8sT/Policies.go

```diff
@@ -545,18 +545,15 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 			}
 		}
 		It("checks policy example", func() {
+
 			waitforPods()
 
 			By("Apply policy to web")
-			eps := kubectl.CiliumEndpointPolicyVersion(ciliumPod)
-			_, err := kubectl.CiliumPolicyAction(
+			_, err = kubectl.CiliumPolicyAction(
 				helpers.KubeSystemNamespace, kubectl.ManifestGet(webPolicy),
 				helpers.KubectlApply, 300)
 			Expect(err).Should(BeNil(), "Cannot apply web-policy")
 
-			err = helpers.WaitUntilEndpointUpdates(ciliumPod, eps, 3, kubectl)
```

Comment:
Why remove this? Aren't the endpoints ready at this time?

Reply:
`CiliumPolicyAction` already waits, so there's no point in waiting again:

https://github.com/cilium/cilium/blob/cdfb40c8aa4c26b83061555984d9a65788fdb2ae/test/helpers/kubectl.go#L690